.pipelines/pipeline.yaml

```diff
@@ -159,32 +158,18 @@ stages:
           name: "$(BUILD_POOL_NAME_LINUX_AMD64)"
         strategy:
           matrix:
-            cni_dropgz_linux_amd64:
+            cni_linux_amd64:
               arch: amd64
-              name: cni-dropgz
+              name: cni
               os: linux
-            cni_dropgz_test_linux_amd64:
```

Comment:
Yay I'm glad we're removing these "dropgz_test"s now, it's much clearer
What I understand, is these "test" targets were made to incorporate a CNI binary from the same build as the dropgz image
But now with this change, I believe we always get that, so no need for these "test" targets anymore

Reply:
exactly